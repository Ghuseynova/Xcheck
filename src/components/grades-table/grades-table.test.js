import React from 'react';
import {render} from "@testing-library/react";
import GradesTable from './grades-table';
import tableData from './test-data';

describe('render', () => {    
  test('Grades Table', () => {
  const result = render(<GradesTable tableData={tableData}/>);
    expect(result).toMatchSnapshot()
  });
});