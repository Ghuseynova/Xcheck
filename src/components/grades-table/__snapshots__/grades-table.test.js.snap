// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Grades Table 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-table-wrapper"
      >
        <div
          class="ant-spin-nested-loading"
        >
          <div
            class="ant-spin-container"
          >
            <div
              class="ant-table ant-table-middle ant-table-bordered"
            >
              <div
                class="ant-table-title"
              >
                task-1-bp1
              </div>
              <div
                class="ant-table-container"
              >
                <div
                  class="ant-table-content"
                >
                  <table
                    style="table-layout: auto;"
                  >
                    <colgroup>
                      <col
                        style="width: 150px; min-width: 150px;"
                      />
                      <col
                        style="width: 450px; min-width: 450px;"
                      />
                    </colgroup>
                    <thead
                      class="ant-table-thead"
                    >
                      <tr>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Score
                        </th>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Comment
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      class="ant-table-tbody"
                    >
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="0"
                      >
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          15
                        </td>
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Well done!
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-table-wrapper"
      >
        <div
          class="ant-spin-nested-loading"
        >
          <div
            class="ant-spin-container"
          >
            <div
              class="ant-table ant-table-middle ant-table-bordered"
            >
              <div
                class="ant-table-title"
              >
                task-1-ep1
              </div>
              <div
                class="ant-table-container"
              >
                <div
                  class="ant-table-content"
                >
                  <table
                    style="table-layout: auto;"
                  >
                    <colgroup>
                      <col
                        style="width: 150px; min-width: 150px;"
                      />
                      <col
                        style="width: 450px; min-width: 450px;"
                      />
                    </colgroup>
                    <thead
                      class="ant-table-thead"
                    >
                      <tr>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Score
                        </th>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Comment
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      class="ant-table-tbody"
                    >
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="1"
                      >
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          0
                        </td>
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Some things are done, some are not
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-table-wrapper"
      >
        <div
          class="ant-spin-nested-loading"
        >
          <div
            class="ant-spin-container"
          >
            <div
              class="ant-table ant-table-middle ant-table-bordered"
            >
              <div
                class="ant-table-title"
              >
                task-1-fp1
              </div>
              <div
                class="ant-table-container"
              >
                <div
                  class="ant-table-content"
                >
                  <table
                    style="table-layout: auto;"
                  >
                    <colgroup>
                      <col
                        style="width: 150px; min-width: 150px;"
                      />
                      <col
                        style="width: 450px; min-width: 450px;"
                      />
                    </colgroup>
                    <thead
                      class="ant-table-thead"
                    >
                      <tr>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Score
                        </th>
                        <th
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          Comment
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      class="ant-table-tbody"
                    >
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="2"
                      >
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          -20
                        </td>
                        <td
                          class="ant-table-cell"
                          style="text-align: center;"
                        >
                          No ticket today
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-table-wrapper"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table ant-table-middle ant-table-bordered"
          >
            <div
              class="ant-table-title"
            >
              task-1-bp1
            </div>
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup>
                    <col
                      style="width: 150px; min-width: 150px;"
                    />
                    <col
                      style="width: 450px; min-width: 450px;"
                    />
                  </colgroup>
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Score
                      </th>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Comment
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="0"
                    >
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        15
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Well done!
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-table-wrapper"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table ant-table-middle ant-table-bordered"
          >
            <div
              class="ant-table-title"
            >
              task-1-ep1
            </div>
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup>
                    <col
                      style="width: 150px; min-width: 150px;"
                    />
                    <col
                      style="width: 450px; min-width: 450px;"
                    />
                  </colgroup>
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Score
                      </th>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Comment
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="1"
                    >
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        0
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Some things are done, some are not
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-table-wrapper"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table ant-table-middle ant-table-bordered"
          >
            <div
              class="ant-table-title"
            >
              task-1-fp1
            </div>
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup>
                    <col
                      style="width: 150px; min-width: 150px;"
                    />
                    <col
                      style="width: 450px; min-width: 450px;"
                    />
                  </colgroup>
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Score
                      </th>
                      <th
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        Comment
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="2"
                    >
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        -20
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: center;"
                      >
                        No ticket today
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
